node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -0,0 +1,97 @@
+#![allow(missing_docs)]
+use cumulus_primitives_core::ParaId;
+use frequency_runtime::{AccountId, AuraId, SudoConfig, EXISTENTIAL_DEPOSIT};
+use sc_service::ChainType;
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec = sc_service::GenericChainSpec<frequency_runtime::GenesisConfig, Extensions>;
+
+/// The default XCM version to set in genesis config.
+const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
+
+use super::{get_properties, Extensions};
+
+pub fn frequency_rococo_testnet() -> ChainSpec {
+	let properties = get_properties("UNIT", 12, 42);
+	let para_id: ParaId = 4044.into();
+	ChainSpec::from_genesis(
+		// Name
+		"Frequency",
+		// ID
+		"frequency_rococo",
+		ChainType::Live,
+		move || {
+			frequency_rococo_genesis(
+				// initial collators.
+				vec![],
+				None,
+				vec![],
+				para_id,
+			)
```

Comment:
@wilwade  we need to discuss or figure out configuration for genesis Kilt is a good example
https://github.com/KILTprotocol/kilt-node/blob/bbacd9160fe637af263ae18bebede3ec34a1c7d2/nodes/parachain/src/chain_spec/spiritnet.rs#L242

Reply:
We will need to do bare minimum configuration like collators, accounts etc. as we would not want to use public accounts , keys etc.

Reply:
@saraswatpuneet 

Some details:

- MaxCandidates can be set to 0 so as it is only the invulnerable collators for now.
- I think we just need 2 collators for now `MaxInvulnerables = 2`
- Work with Jacob to generate the secrets that he will use for the two collators
- For now we can just stick to Sudo until #248 then we can use that, except still with one or two keys.
- Use `16CnNyH6a6wPr5pH8GJ3AyRP1fJN2MDJxKUJtedX1Zd4zW7i` for sudo for now.
- Make sure to fund sudo ;)
- I think that gets everything we have to have for initial tests right?

Reply:
Yep i think this should be enough for me to get testnet up as early as next week when I will work with @jacobmfoley to get collator secrets
Thanks!

Reply:
Tagging #270 #271 and #273 as related.